go/eks/simple-cluster/constructs/addons/load-balancer-controller.go

```diff
@@ -0,0 +1,321 @@
+package addons
+
+import (
+	"github.com/aws/aws-cdk-go/awscdk/v2"
+	"github.com/aws/aws-cdk-go/awscdk/v2/awseks"
+	"github.com/aws/aws-cdk-go/awscdk/v2/awsiam"
+	"github.com/aws/jsii-runtime-go"
+)
+
+// Install AWS Load Balancer Controller
+// https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html
+func NewEksLoadBalancerController(stack awscdk.Stack, cluster awseks.Cluster) {
```

Comment:
Fundamentally, this needs to be scoped down at least to some degree. The permissions defined in these IAM policies are too broad and feel too expansive.

Is there a reason this isn't scoped down to individual resources? Could this be reduced to a smaller scope?